app/views/devise/shared/_links.html.erb

```diff
@@ -3,7 +3,7 @@
 <% end -%>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+  <%= link_to "New to PlaceMe!", new_registration_path(resource_name) %><br />
```

Comment:
I need some help with this. This might be the way to create translated links no?. Where can I find the options available for use or how can I construct each link?